h/form.py

```diff
@@ -75,7 +82,7 @@ def create_form(request, *args, **kwargs):
     default) will use the renderer configured in the :py:mod:`h.form` module.
     """
     env = request.registry[ENVIRONMENT_KEY]
-    renderer = Jinja2Renderer(env, {"feature": request.feature})
+    renderer = Jinja2Renderer(env, {"feature": request.feature, "request": request})
```

Comment:
Passing `request` to the renderer so it can add it to the template context above.